Cpix/CpixDocument.cs

```diff
@@ -199,7 +237,7 @@ public void Save(Stream stream)
 				{
 					// NB! Do not apply any formatting here, as digital signatures have already been generated
 					// and any formatting will invalidate the signatures!
-					Encoding = Encoding.UTF8,
+					Encoding = new UTF8Encoding(false),
```

Comment:
Perhaps a comment that "false" is used to stop a BOM from being emitted?

Reply:
Used this style:

Encoding = new UTF8Encoding(encoderShouldEmitUTF8Identifier:false)